FRE/fre_make.json

```diff
@@ -0,0 +1,202 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "name": {
+            "description": "The name of the experiment",
+            "type": "string"
+        },
+        "platform": {
+            "description": "The platforms listed in the command",
+            "type": "string"
+        },
+        "target": {
+            "description": "The targets listed in the command",
+            "type": "string"
+        },
+        "build": {
+           "type": "object",
+           "additionalProperties": false,
+           "properties": {
+               "compileYaml": {
+                   "description": "Path to the compile yaml.",
+                   "type": "string"
+               },
+               "platformYaml": {
+                   "description": "Path to the platform yaml.",
+                   "type": "string"
+               }
+           }
+        },
+        "compile": {
+            "description": "The source code descriptions",
+            "$ref": "#/definitions/Compile"
+        },
+        "platforms": {
+            "description": "FRE platforms",
+            "type": "array",
+            "items": {"$ref": "#/definitions/Platform"}
+        }
+    },
+    "definitions": {  
+        "Compile": {
+            "type": "object",
+            "properties": {
+                "experiment": {
+                    "description": "The name of the model",
+                    "type": "string"
+                },
+                "container_addlibs": {
+                    "description": "Libraries and packages needed for linking in the container",
+                    "type": ["array","null"],
+                    "items": {"type": "string"}
```

Comment:
`items` wouldn't be applicable in the case where the type is `null`. While this may be fine (it's unclear to me whether or not superfluous keywords are allowed), I think separating the `null` and `array` cases using `oneOf` might be more clear:

```
oneOf: [
  {"type": "null"},
  {
    "type": "array",
    "items": {"type": "string"}
  }
]
```

Reply:
Ok, so would this also apply to `baremetal_linkerflags` and `RUNenv` then?

Reply:
Yep, `container_addlibs`, `baremetal_linkerflags`, and `RUNenv` are the three that I'm noticing.

Reply:
Oh whoops, I don't think `RUNenv` should even have `null`.